### DISCUSSION TITLE
Poll – Multi-Monitor and Layout-Mode usage

### DISCUSSION THREAD

**Discussion by hertg** [leftwm/leftwm#767] [Polls]
Are you using a multi-monitor (or multi-workspace) setup? And which [layout mode](https://github.com/leftwm/leftwm/wiki/Config#layout-mode) do you prefer?
*Hint: If you haven't configured it explicitly, the `LayoutMode` defaults to `Workspace`.*

**Comment by Kirirou** [comment]
I like the feature of separate tags on each monitor
